src/app/questionnaire-item/store/transform-initial-state.ts

```diff
@@ -230,7 +230,10 @@ const getExistingTypeProperty: (
       R.hasPath([`${prefix}Coding`, 'code']),
       R.path([`${prefix}Coding`, 'code']),
     ],
-    [R.has(`${prefix}Quantity`), R.prop(`${prefix}Quantity`)],
+    [
+      R.hasPath([`${prefix}Quantity`, 'value']),
+      R.path([`${prefix}Quantity`, 'value']),
+    ],
```

Comment:
Is this change just from the definition? Seems a lot different than most other definitions.

Reply:
`valueQuantity` is an object (https://www.hl7.org/fhir/datatypes.html#Quantity) and not a primitive type as most of the other cases there.